clang/include/clang/CIRFrontendAction/CIRGenAction.h

```diff
@@ -0,0 +1,61 @@
+//===---- CIRGenAction.h - CIR Code Generation Frontend Action -*- C++ -*--===//
```

Comment:
Should we move this header to `CIR` or `FrontendAction`? Currently it lives in `CIRFrontendAction` but its implementation file lives in `FrontendAction`

Reply:
Good eye, that was a typo I guess.